src/clr-angular/utils/a11y/aria-live.service.ts

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { PLATFORM_ID } from '@angular/core';
+import { DOCUMENT, isPlatformBrowser } from '@angular/common';
+import { Inject, Injectable, NgZone, OnDestroy } from '@angular/core';
+import { uniqueIdFactory } from '../id-generator/id-generator.service';
+
+export enum AriaLivePoliteness {
+  off = 'off',
+  polite = 'polite',
+  assertive = 'assertive',
+}
+
+/**
+ * Time in milliseconds before inserting the content into the container
+ */
+export const ARIA_LIVE_TICK: number = 100;
+
+@Injectable()
+export class AriaLiveService implements OnDestroy {
+  private ariaLiveElement: HTMLElement;
+  private document: Document;
+  private previousTimeout: ReturnType<typeof setTimeout>;
+  private id: string = `clr-aria-live-element-${uniqueIdFactory()}`;
+
+  constructor(
+    private _ngZone: NgZone,
+    @Inject(DOCUMENT) _document: any,
+    @Inject(PLATFORM_ID) private platformId: Object
+  ) {
+    this.document = _document;
+  }
+
+  private createLiveAriaContainer(): HTMLElement {
```

Comment:
We have two methods that have almost the identical names, `createLiveContainer` 
 and `createLiveAriaContainer`, why?

Also, this method should handle inside itself whether it should create live aria container element or just exit if there is an already existing aria-live element.